pkg/cmd/pulumi/policy/policy_publish.go

```diff
@@ -138,15 +146,10 @@ func (cmd *policyPublishCmd) Run(ctx context.Context, lm cmdBackend.LoginManager
 	return nil
 }
 
-func requirePolicyPack(
+func loginToCloudBackend(
```

Comment:
Please minimize unrelated diffs

Reply:
ahh this isn't unrelated, this is a light refactor to expose the backend for both looking up the default org and for getting the current policy pack, to reduce code duplication

this commit is an easier view:
https://github.com/pulumi/pulumi/pull/19249/commits/212a27c69af1902f550b9a078f4b1d412b66a4e1